orchestra/workflow/certifications.py

```diff
@@ -0,0 +1,30 @@
+from django.db import transaction
+
+from orchestra.models import Certification
+from orchestra.models import WorkerCertification
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+@transaction.atomic
+def migrate_certifications(workflow_src, workflow_dst, certification_slugs):
```

Comment:
you know I'm going to tell you to spell these out:
* workflow_src => source_workflow
* workflow_dst => destination_workflow
* old_worker_certs => old_worker_certifications